cmd/zoekt-sourcegraph-indexserver/cleanup.go

```diff
@@ -205,6 +207,7 @@ func moveAll(dstDir string, shards []shard) {
 			removeAll(shards)
 			return
 		}
+		_ = os.Rename(shard.Path+".meta", dst+".meta")
```

Comment:
What makes it OK to ignore this error?

Reply:
This PR is a POC and not ready for merging so this is the shortcut for the POC :) Will make this good soon.